test/test-helper.el

```diff
@@ -3,7 +3,8 @@
 (require 'f)
 (let ((elpy-dir (f-parent (f-dirname (f-this-file)))))
   (add-to-list 'load-path elpy-dir)
-  (add-to-list 'process-environment (format "PYTHONPATH=%s" elpy-dir))
+  (add-to-list 'process-environment (format "PYTHONPATH=%s:%s" elpy-dir
```

Comment:
Nitpick, could you put `elpy-dir` on a new line for readability? :-)

Reply:
Hello Jorgen! Sure! Update incoming.